environs/__init__.py

```diff
@@ -176,6 +177,16 @@ def _preprocess_json(value: str, **kwargs):
     return pyjson.loads(value)
 
 
+_EnumT = typing.TypeVar("_EnumT", bound=Enum)
+
+
+def _enum_parser(value, *, type: typing.Type[_EnumT], **kwargs) -> _EnumT:
+    try:
+        return type[value]
+    except Exception:
+        raise ma.ValidationError(f"Not a valid {type} enum.")
```

Comment:
with the suggested change above, we can do `raise ... from ...`:

```suggestion
        raise ma.ValidationError(f"Not a valid {type} enum.") from error
```

Reply:
No need to do `from error` in an `except` clause as explained in this SO answer https://stackoverflow.com/a/24752607/5437493